extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -14,4 +14,63 @@ public static function getHTMLBeforeWikiaPage( &$html ) {
 		}
 		return true;
 	}
+
+	/**
+	 * Cache key invalidation when an article is edited
+	 *
+	 * @param $article
+	 * @param $user
+	 * @param $text
+	 * @param $summary
+	 * @param $minoredit
+	 * @param $watchthis
+	 * @param $sectionanchor
+	 * @param $flags
+	 * @param $revision
+	 * @param $status
+	 * @param $baseRevId
+	 * @return bool
+	 */
+	public static function onArticleSaveComplete(
+		&$article, &$user, $text, $summary, $minoredit, $watchthis,
+		$sectionanchor, &$flags, $revision, &$status, $baseRevId ) {
```

Comment:
On multiline definitions like this, we switch to Allman-style braces to make the separation between the definition and the body clearer. i.e.
```php
public static function onArticleSaveComplete(
	$article, $user, $text, $summary, $minoredit, $watchthis,
	$sectionanchor, &$flags, $revision, &$status, $baseRevId
) {
	// Function body
}
```